### PR TITLE
Feature/ele 826 create new planning edit

### PR DESCRIPTION
**Pull request by gularsson**
Requires https://github.com/ttab/elephant-ui/pull/454

### CODE REVIEW

File: src/components/Description.tsx [hunk 1]
```diff
@@ -13,12 +13,12 @@ export const Description = ({ role }: {
   const path = `meta.core/description[${index}].data.text`
 
   return (
-    <div className='flex w-full -ml-1' >
+    <div className='flex w-full' >
```
Comment (Victor-Lindh) [inline src/components/Description.tsx:16]: Micro trailing space

File: src/views/Planning/index.tsx [hunk 2]
```diff
@@ -138,76 +129,50 @@ const PlanningViewContent = (props: ViewProps & { documentId: string }): JSX.Ele
...
-              />
-              <div className='min-w-32'>
-                <SluglineEditable
-                  path='meta.tt/slugline[0].value'
-                  documentStatus={documentStatus?.name}
-                  onValidation={handleValidation}
-                />
-              </div>
-            </div>
+                placeholder='Planeringtitel'
```
Comment (Victor-Lindh) [inline src/views/Planning/index.tsx:137]: Should be `Planeringstitel` ?

File: src/components/Section.tsx [hunk 3]
```diff
@@ -22,41 +21,41 @@ export const Section = ({ onValidation }: {
...
+      >
+        <ComboBox
+          max={1}
+          size='xs'
+          modal={true}
+          sortOrder='label'
+          options={allSections}
+          selectedOptions={selectedOptions}
+          placeholder={section?.title || 'Lägg till sektion'}
+          validation={!!onValidation}
```
Comment (Victor-Lindh) [inline src/components/Section.tsx:39]: `Property 'validation' does not exist on type 'IntrinsicAttributes & ComboBoxControlledProps'.`

Comment (gularsson) [reply src/components/Section.tsx:39]: Get the latest, install latest @ttab/elephant-ui and it should work :crossed_fingers:

**Review by dannelundqvist — APPROVED** [review]

**Pull request merged**